### PR TITLE
New test diffuse 4822

### PR DESCRIPTION
**Pull request by giorgio-pirola**
# New Prod config

### CODE REVIEW

File: production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+prod_type: TestDiffuseGammas
+stages_to_run:
+- dl1ab
+- merge_dl1
+- dl1_to_dl2
+- dl2_to_irfs
+stages:
+  dl1ab:
+  - input: /fefs/aswg/data/mc/DL1/AllSky/20220527_src2_diffgamma/TestingDataset/GammaDiffuse/node_theta_14.984_az_355.158_
+    output: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/node_theta_14.984_az_355.158_
```
Comment (vuillaut) [inline production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml:31]: the output date formatting is wrong
please use `YYYYMMDD` to be consistent with all other prods

File: production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  - input: /fefs/aswg/data/mc/DL1/AllSky/20220527_src2_diffgamma/TestingDataset/GammaDiffuse/node_theta_52.374_az_301.217_
+    output: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/node_theta_52.374_az_301.217_
+  - input: /fefs/aswg/data/mc/DL1/AllSky/20220527_src2_diffgamma/TestingDataset/GammaDiffuse/node_theta_32.059_az_355.158_
+    output: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/node_theta_32.059_az_355.158_
+  - input: /fefs/aswg/data/mc/DL1/AllSky/20220527_src2_diffgamma/TestingDataset/GammaDiffuse/node_theta_43.197_az_262.712_
+    output: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/node_theta_43.197_az_262.712_
+  merge_dl1:
+  - input: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/node_theta_14.984_az_355.158_
+    output: /fefs/aswg/data/mc/DL1/AllSky/24052022_dec_4822/TestingDataset/GammaDiffuse/dl1_24_05_2022_dec_4822_node_theta_14.984_az_355.158__merged.h5
```
Comment (vuillaut) [inline production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml:40]: Same comment here, some consistency in the date would be better

File: production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# lstmcpipe generated config from PathConfigAllSkyFullDL1ab - 2022-05-24
+
+workflow_kind: lstchain
+
+# prod_id ex: local_no_n_islands. Default; v00 (if key left empty or None)
+prod_id: 24052022_dec_4822
```
Comment (vuillaut) [inline production_configs/20220524_dec_4822_tuned_TestDIffuseGammas/lstmcpipe_config_2022-05-24_TestDiffuseGammas.yaml:6]: change to `20220524_dec_4822_tuned`

**Review by vuillaut — CHANGES_REQUESTED** [review]
Please change the date formatting everywhere to be consistent with other prods and list them by date order correctly

**Comment by vuillaut** [comment]
Other comments:
- When opening the pull-request, the template should be filled with just a some info to help the reviewer (same info that is in the readme, so a simple copy/pasting should do).

- does this PR replaces #327 ?

**Comment by giorgio-pirola** [comment]
> Other comments:
> 
> * When opening the pull-request, the template should be filled with just a some info to help the reviewer (same info that is in the readme, so a simple copy/pasting should do).
> * does this PR replaces [Test diffuse gammas 4822 #327](https://github.com/cta-observatory/lstmcpipe/pull/327) ?

Yes sorry, I opened 3 pull requests in total, but I'd like the last one "New test diffuse 4822" to replace the previous ones!

**Comment by garciagenrique** [comment]
> > Other comments:
> > 
> > * When opening the pull-request, the template should be filled with just a some info to help the reviewer (same info that is in the readme, so a simple copy/pasting should do).
> > * does this PR replaces [Test diffuse gammas 4822 #327](https://github.com/cta-observatory/lstmcpipe/pull/327) ?
> 
> Yes sorry, I opened 3 pull requests in total, but I'd like the last one "New test diffuse 4822" to replace the previous ones!

Then close the previous PRs @giorgio-pirola , please

**Review by vuillaut — APPROVED** [review]

**Comment by giorgio-pirola** [comment]
> I don't know why the tests are still failing, valid configs. Will work on the tests later, I merge.

Should I delete the branch?

**Comment by vuillaut** [comment]
@giorgio-pirola I have fixed some issues with the config (see #329 and #330) and launched the prod.
However, I just realised you are producing IRFs with diffuse gammas without any background, is that really useful for high-level analysis?

**Comment by vuillaut** [comment]
I had to fix a lot of tiny things and typos but I finally got the prod done:
```
/fefs/aswg/data/mc/{data_level}/AllSky/20220624_dec_4822_tuned/TestingDataset/dec_4822/
```

**Pull request merged**